### PR TITLE
Updated draft text to be more clear as to how to participate in RFC

### PR DESCRIPTION
**Pull request by realrasengan**

### CODE REVIEW

**Comment by hanetzer** [comment]
redirecting discussion to another issue doesn't stop the fact you did a shitty thing, got caught on it, and are now retroactively changing policy to make what you did 'ok'.

**Comment by marksteward** [comment]
Not just that, it was a minor shitty thing that might well have been justified, but instead of discussing it openly, you've retconned this policy change. The new wording doesn't even apply to the case of `##hntop` because it's not a project channel, but it sure look like an attempt to justify it.

The diff in #513 has three effects:
 - it allows for duplicate project channels on Freenode when those projects have moved to other networks
 - it enlarges the scope of off-topic (i.e. allowed under sufferance) channels to include listable channels that have moved and "squatting a namespace"
 - it changes the scope of off-topic (i.e. allowed under sufferance) channels to include "illicit content" instead of "heavy media file trading, proprietary game software modding, porn and various forms of antisocial behaviour, including (but not limited to) political, racial, ethnic, religious or gender-related invective"

This was always a badly worded part of the policy. It was probably meant to say "... are not welcome on Freenode" instead of "... are considered off-topic", and that's how people are reading this change. It's as close as Freenode had to an anti-harassment policy, and you've removed the part of it that mentioned political, racial, ethnic, religious and gender-related invective.

The remaining text only says that you don't "endorse or encourage discrimination". That sends a pretty strong signal about what the new opers will be prioritising. If this isn't a signal to free-speech maximalists and abusers, it's a spectacular misfire.

**Comment by hyder365** [comment]
wtf i love freenode now

**Comment by Be-ing** [comment]
Good to know Freenode is run by "free speech" extremists now!

**Comment by wizzwizz4** [comment]
@Be-ing Avoid trolling, please. There's enough to criticise here – such as the precedent set by an “I am the Senate”-type move – without resorting to such indirection as “this is a move that the bad people like”.

That being said, I would appreciate a stronger justification for the removal without replacement of the antisocial behaviour clause. This significantly reduces the admin purview to crack down on cross-channel harassment (unless, of course, the rules get revised again shortly after the admins decide to deal with a case of it).

**Comment by Be-ing** [comment]
From recent events, #513, and this PR, it is clear to me these changes are not being made in good faith so I'm not going to engage with them as if they are. If you want to spend, and probably waste, your time communicating with these people now running Freenode, that's up to you, but this is the last I'm commenting on it.

File: content/pages/policies.md [hunk 1]
```diff
@@ -43,7 +43,8 @@ Please contact network staff if you would like to take over an expired nickname.
 Channel ownership
 =================
 
-*DRAFT*
+*These changes are made on 5/22/21 and are a draft. Please send feedback to
```
Comment (nemobis) [inline content/pages/policies.md:46]: https://xkcd.com/1179/

**Comment by kakolisgay** [comment]
I have no idea why everyone is upset but I find it very amusing

**Comment by BurnyBoi** [comment]
Good thing I use Matrix!

File: content/pages/policies.md (hunk 1 above)
Comment (morsik) [reply content/pages/policies.md:46]: Yes, I don't know name for 22nd month in year… I didn't even know that there are more months than 12 :P

**Comment by yardenac** [comment]
With [this comment](https://github.com/freenode/web-7.0/pull/513/commits/2037126831a84c57f978268f090fc663cf43ed7a#r637536729) in #513, @dhjw justifies removing language about bigoted invective on freenode by saying "The answer to undesirable speech is more speech, not suppression". I wanted to reply to that comment, but was unable to, so I'm posting my reply here.

While the logic is tempting, the "add more speech" approach can become futile in practice, given the bullshit asymmetry. IME most functioning communities balance free speech as one virtue among many. Freenode, however, seems to now be leaning in the direction of free speech extremism. This will mostly benefit manipulative sociopaths such as the nazis who use irc to actively recruit. It will not benefit the people who mostly use irc to run software projects. They'll be drowned out in the noise, and will find it much harder to maintain diverse and inclusive communities on freenode.

**Comment by nemobis** [comment]
Il 24/05/21 22:45, Yardena Cohen ha scritto:
> "The answer to undesirable speech is more speech, not suppression"

Not everyone agrees, naturally, but this is a very standard argument. 
See for instance Nadine Strossen:
https://openlibrary.org/works/OL19664268W/Hate

I don't think we need to revisit this entire matter here.

**Comment by kakolisgay** [comment]
> @yardenac While the logic is tempting, the "add more speech" approach can become futile in practice

If you can't counter your opposition on a level playing field perhaps you should reevaluate your position. For software project channels, politics shouldn't be the topic of discussion anyway.

> This will mostly benefit manipulative sociopaths such as the nazis who use irc to actively recruit.

"The nazis" have been gone for decades. The only place they continue to exist is living rent free in your head.

**Comment by yardenac** [comment]
There are, in fact, nazis on freenode. If leadership is in denial, the problem will only get worse.

A level playing field implies specific goalposts, and I'm not naive enough to try to persuade ideologues who've made up their minds as thoroughly as the new freenode regime. At least, not until a few years have passed and the humblest 20% of you realize that your community is still harmful and awful.

**Comment by dhjw** [comment]
As I understand, some or all of Freenode servers fall under UK law, which provides numerous prohibitions against harassing behaviour. See [this wiki](https://en.wikipedia.org/wiki/Harassment_in_the_United_Kingdom#Legislation). So people don't actually have to worry about the removal of vague and subjective wording from the policy, or even staffer inaction, because police are employed to find and punish such things.

**Comment by hanetzer** [comment]
gentoo-proxy-maint
Note that freenode is now forcing topic changes for massive channels who have decided to make the move to libera.
freenodecom has changed topic for #gentoo-proxy-maint from "https://www.gentoo.org/news/2021/05/23/Moving-to-Libera.html | #gentoo-proxy-maint is moving to libera.chat. Please join us in #gentoo-proxy-maint on irc.libera.chat" to "This channel has moved to ##gentoo-proxy-maint. The topic is in violation of freenode policy: https://freenode.net/policies"`
You will not win this.

**Comment by dhjw** [comment]
What's to win, it's their network they can do what they want.

**Comment by morsik** [comment]
Now, this behaviour is real joke. Looks like it was good decision by everybody to move out from Freenode.

And we are just sad about this.

**Comment by Ikfes** [comment]
I can't wait to talk about white genocide and KKK in my favorite channels now.. Im currently speeding home from work.
/Sarcasm

On a serious note, this is tragicomically funny update.

**Comment by dhjw** [comment]
It's not like staff were in every channel enforcing everything before. It was always vague, subjective, random and often abusive. Consider now there's an actual legal entity that can be held legally responsible rather than some disparate collection of anons.

**Pull request merged**